x/ccv/provider/keeper/msg_server.go

```diff
@@ -54,7 +54,7 @@ func (k msgServer) AssignConsumerKey(goCtx context.Context, msg *types.MsgAssign
 	k.Logger(ctx).Info("assigned consumer key",
 		"consumer chainID", msg.ChainId,
 		"validator operator addr", msg.ProviderAddr,
-		"consumer tm pubkey", consumerTMPublicKey.String(),
+		"consumer public key", msg.ConsumerKey,
```

Comment:
Before the public key was logged as something similar to this:
```
5:16PM INF assigned consumer key consumer chainID=neutron-1 consumer tm pubkey="ed25519:\"\\242\\263!y\\256\\304\\246\\023\\031\\360\\323\\215\\337\\263\\020\\271\\203A\\253\\211hk\\205\\356'\\234\\310W>B7\\213\" " module=x/ibc-provider validator operator addr=cosmosvaloper1jmykcq8gylmy5tgqtel4xj4q62fdt49sl584xd
```
We fix this so we get the public key logged as:
```
`{"@type":"/cosmos.crypto.ed25519.PubKey","key":"Ui5Gf1+mtWUdH8u3xlmzdKID+F3PK0sfXZ73GZ6q6is="}
```

Thanks to @MSalopek for pointing this out!